### PR TITLE
Add assimp

### PR DESCRIPTION
**Pull request by PucklaMotzer09**
Adds the assimp library.

### CODE REVIEW

File: packages/a/assimp/xmake.lua [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            add_config_arg("build_assimp_tools", "ASSIMP_BUILD_ASSIMP_TOOLS")
+        end
+
+        if is_plat("android", "iphoneos") then
+            table.insert(config, "-DASSIMP_BUILD_ASSIMP_TOOLS=OFF")
+        end
+
+        import("package.tools.cmake").install(package, config)
+        if is_plat("linux") then
+            package:add("links", "assimp", "IrrXML")
```
Comment (waruqi) [inline packages/a/assimp/xmake.lua:95]: you can uses add_syslinks("irrXML") to ensure link order.

**Review by waruqi — COMMENTED** [review]
Are you sure all linux systems have built-in irrXML library?

**Comment by waruqi** [comment]
In order to ensure that xmake-repo does not have a long commit history, please merge all the commits into single commit.

Thanks.

File: packages/a/assimp/xmake.lua (hunk 1 above)
Comment (PucklaMotzer09) [reply packages/a/assimp/xmake.lua:95]: The irrXML library is built into assimp and gets compiled with it when ```system_irrxml``` is false. Which is the default behaviour.

**Comment by waruqi** [comment]
I have merged into the master branch.

**Pull request merged**